packages/cli/tests/build.test.js

```diff
@@ -136,4 +137,11 @@ describe('preact build', () => {
 		let dir = await subject('location-patch');
 		expect(() => build(dir)).not.toThrow();
 	});
+
+	it('should copy resources from static/root to root directory', async () => {
+		let dir = await subject('static-root');
+		await build(dir);
+		let file = join(dir, '.htaccess');
```

Comment:
Static root files should still be placed into the build output directory:

```suggestion
		let file = join(dir, 'build', '.htaccess');
```